_pages/relayimps/Minds Nostr Relay.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Minds Nostr Relay
+permalink: /Minds Nostr Relay/
+web: 
+github: https://gitlab.com/minds/infrastructure/nostr-relay
```

Comment:
This always has to be only account/repo!

```suggestion
github: infrastructure/nostr-relay
```

Reply:
ok, error appears because it is gitlab. changing...